.github/workflows/pr_test.yml

```
@@ -60,9 +60,161 @@ on:
+          git rebase origin/${{ env.BASE_BRANCH }}
+
+      - name: Merge
+        working-directory: ${{ env.PS_DIR }}
+        if: ${{ env.REBASE_OR_MERGE == 'merge' }}
+        run: |
+          git fetch origin ${{ env.BASE_BRANCH }}:${{ env.BASE_BRANCH }}
+          git merge origin/${{ env.BASE_BRANCH }}
+
+      # Certificate
+      - name: Generate a certificate
+        if: (env.BASE_BRANCH == '8.1.x') || (env.BASE_BRANCH == 'develop')
+        run: |
+          ## Install MkCert
+          sudo apt install libnss3-tools
+          curl -JLO "https://dl.filippo.io/mkcert/latest?for=linux/amd64"
+          chmod +x mkcert-v*-linux-amd64
+          sudo cp mkcert-v*-linux-amd64 /usr/local/bin/mkcert
+          ## Generate certificate
+          mkcert -key-file ./${{ env.PS_DIR }}/.docker/ssl.key -cert-file ./${{ env.PS_DIR }}/.docker/ssl.crt localhost
+          ## Link certificate to Chrome Trust Store
+          mkdir -p $HOME/.pki/nssdb
+          certutil -d $HOME/.pki/nssdb -N
+          certutil -d sql:$HOME/.pki/nssdb -n localhost -A -t "TCu,Cu,Tu" -i ./${{ env.PS_DIR }}/.docker/ssl.crt
+          ## Add self-signed certificate to Chrome Trust Store
+          mkcert -install
+
+      # Workaround until https://github.com/PrestaShop/PrestaShop/issues/29813 is fixed
+      - name: PrestaShop Configuration (Copy of Config API)
+        if: (env.BASE_BRANCH == '8.1.x') || (env.BASE_BRANCH == 'develop')
+        run: cp ./${{ env.PS_DIR }}/app/config/security_test.yml ./${{ env.PS_DIR }}/app/config/security_prod.yml
+
+      - name: Adapt Back Office layout
+        if: inputs.backoffice_layout == 'symfony'
+        working-directory: ${{ env.PS_DIR }}
+        shell: bash
+        run: |
+          echo PS_FF_SYMFONY_LAYOUT=true >> .env
+
+      # UI Tests : Setup, Install & Execute
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Setup NPM
+        run: npm install -g npm@7
+
+      - name: Install tests dependencies
+        run: npm install
+        working-directory: '${{ env.PS_DIR }}/tests/UI'
+
+      # Create shop with Docker build assets, and initialize database and shop content
+      - name: Build and run shop with docker
+        working-directory: ${{ env.PS_DIR }}
+        timeout-minutes: 15
+        env:
+          URL_FO: ${{ ((github.event.inputs.base_branch == '8.0.x') || (github.event.inputs.base_branch == '1.7.8.x')) && 'http://localhost:8001/' || 'https://localhost:8002/' }}
+          VERSION: ${{ (github.event.inputs.base_branch == '1.7.8.x') && github.event.inputs.php_version || env.VERSION }}
+          # Initial build, we make assets and install shop data
+          DISABLE_MAKE: 0
+          PS_INSTALL_AUTO: 1
+        run: |
+          echo Build docker via docker composer
+          USER_ID=$(id -u) GROUP_ID=$(id -g) docker-compose -f docker-compose.yml up -d --build
```

by default it takes the docker-compose.yml file
```suggestion
          USER_ID=$(id -u) GROUP_ID=$(id -g) docker-compose up -d --build
```